docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -443,6 +443,12 @@ So if you want to deploy your application somewhere, you need to copy the _runne
 
 NOTE: Before running the application, don't forget to stop the hot reload mode (hit `CTRL+C`), or you will have a port conflict.
 
+== Configuring the banner
+
+By default when a quarkus application runs (in regular or dev mode), it will display an ASCII art banner. The banner can be disabled by setting `quarkus.banner.enabled=false` in `application.properties`
```

Comment:
```suggestion
By default when a Quarkus application runs (in regular or dev mode), it will display an ASCII art banner. The banner can be disabled by setting `quarkus.banner.enabled=false` in `application.properties`
```